src/js/base/runtime.js

```diff
@@ -1230,7 +1224,6 @@ function (Namespace, jsnums, codePoint, util, exnStackParser, loader, seedrandom
     ************************/
     function checkType(val, test, typeName) {
       if(!test(val)) {
-        debugger;
```

Comment:
I removed these `debugger;` statements since they make running the test-suite in the browser impossible without clicking the "resume" button a whole lot.